src/Sylius/Bundle/OrderBundle/DependencyInjection/Compiler/TagCartContextsPass.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\OrderBundle\DependencyInjection\Compiler;
+
+use Sylius\Component\Order\Context\ResettingCartContextInterface;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class TagCartContextsPass implements CompilerPassInterface
```

Comment:
```suggestion
final class ResettingCartContextsPass implements CompilerPassInterface
```
?
At a glance, I'd assume that `TagCartContextsPass` is responsible for tagging services w/ the `sylius.context.cart` tag instead.